sdks/python/apache_beam/runners/dataflow/internal/apiclient.py

```diff
@@ -478,6 +544,19 @@ def __init__(self, options):
         get_credentials=(not self.google_cloud_options.no_auth),
         http=http_client,
         response_encoding=get_response_encoding())
+    self._sdk_image_overrides = self._get_sdk_image_overrides(options)
+
+  def _get_sdk_image_overrides(self, pipeline_options):
+    worker_options = pipeline_options.view_as(WorkerOptions)
+    sdk_overrides = worker_options.sdk_harness_container_image_overrides
+    overrides_dict = dict()
```

Comment:
Again, one can write

`overrides_dict = dict(override_str.split(',', 1) for override_str in sdk_overrides)`